test/suites/basic.sh

```diff
@@ -41,6 +41,44 @@ test_interactive() {
     validate_system_lxd "${m}" 3 disk1
   done
 
+  # Reset the systems with just LXD and no IPv6 support.
+  reset_systems 3 3 1
+
+  for m in micro01 micro02 micro03 ; do
+    lxc exec "${m}" -- echo 1 > /proc/sys/net/ipv6/conf/all/disable_ipv6
+    lxc exec "${m}" -- snap disable microceph || true
+    lxc exec "${m}" -- snap disable microovn || true
+    lxc exec "${m}" -- snap restart microcloud
+  done
+
+  echo "Creating a MicroCloud with ZFS storage and no IPv6 support"
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+
+  lxc exec micro01 -- tail -1 out | grep "MicroCloud is ready" -q
+  for m in micro01 micro02 micro03 ; do
+    validate_system_lxd "${m}" 3 disk1
+  done
+
+  # Reset the systems with just LXD and no IPv4 support.
+  gw_net_addr=$(lxc network get lxdbr0 ipv4.address)
+  lxc network set lxdbr0 ipv4.address none
+  reset_systems 3 3 1
+
+  for m in micro01 micro02 micro03 ; do
+    lxc exec "${m}" -- snap disable microceph || true
+    lxc exec "${m}" -- snap disable microovn || true
+    lxc exec "${m}" -- snap restart microcloud
+  done
+
+  echo "Creating a MicroCloud with ZFS storage and no IPv4 support"
+  ! microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init 2> out" || false
+
+  # Ensure we exited with the right error message.
+  lxc exec micro01 -- tail -1 out | grep "Error: Could not determine Fan overlay subnet: No default gateway for IPv4" -q
```

Comment:
Ok, I can add a check to skip FAN networking. What about the case where OVN is also not usable? Should we error out here or allow the cluster creation to continue? For example:

```go
if !canOVN {
	fmt.Println("No dedicated uplink interfaces detected, skipping distributed networking")
	if !fanUsable {
		return fmt.Errorf("Neither FAN nor OVN networking is usable")
	}

	return nil
}
```

Reply:
I don't think we need to error out necessarily. But since OVN takes precedence over the FAN network, we shouldn't log anything about the FAN network if we are attempting to set up OVN.

If the cluster size is >= 3 (supports OVN):
* If OVN is unavailable, log that we will skip it
* Then, attempt to fallback to FAN network
* Then, if FAN is unavailable, log that we will skip it

If the cluster size is < 3 (skips OVN entirely):
* If FAN is unavailable, log that we will skip it

Reply:
It can still be set up after the fact. The alternative is that microcloud would not be able to support <3 node systems without ipv4.

Reply:
So from our meeting the other day we concluded that we would add a question for the end user to decide whether or not to proceed with setting up an inoperable cluster (from the point of view of no cluster wide overlay networking being available) in such scenarios.